tests/e2e-leg-11/client-proxy/40-add-subcluster.yaml

```diff
@@ -11,13 +11,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-commands:
-  - command: >
-      kubectl patch verticadb v-client-proxy --type='json' -p='[{
-      "op": "add", 
-      "path": "/spec/subclusters/-", 
-      "value": {"name": "sec1", "size": 2, "type": "secondary"}
-      }]'
-    namespaced: true
+apiVersion: vertica.com/v1
+kind: VerticaDB
+metadata:
+  name: v-client-proxy
+spec:
+  proxy:
+    image: kustomize-vproxy-image
```

Comment:
I don't like this. It will change the proxy image to kustomize-vproxy-image which is an invalid image. Also, make sure the client-proxy test case passes in your local machine first.

Reply:
This should be fine. It allows user to customize their image through the env variable $VPROXY_IMG instead of using the default one.

Reply:
It does noest work as you think, it only works at step 15 when we use `kustomize`. Here it will just update the image to kustomize-vproxy-image which is wrong

Reply:
If we want to use image value from the "kustomize-vproxy-image", what is the proper way to do that?

Reply:
And btw, how is the kustomize-vproxy-image generated? I can't find any other reference of "kustomize-vproxy-image" rather than the setup db yaml

Reply:
@roypaulin you are right. It works only if we run `kustomize build` as step 15. So here we can just don't set image and use the default value.

@HaoYang0000, this is my understanding of how it was generated:
The script scripts/setup-kustomize.sh will put $VPROXY_IMG into the key data.vproxyImage in the e2e configmap. Then each time `kustomize build` will replace kustomize-vproxy-image with that value stored in the key data.vproxyImage. So if the yaml file was not generated by `kustomize build`, kustomize-vproxy-image won't be replaced.